dspace-api/src/main/java/org/dspace/core/Utils.java

```diff
@@ -411,7 +425,50 @@ public static synchronized String formatISO8601Date(Date d)
         return result.substring(0, rl-2) + ":" + result.substring(rl-2);
     }
 
-    public static <E> Collection<E> emptyIfNull(Collection<E> collection) {
+    public static <E> java.util.Collection<E> emptyIfNull(java.util.Collection<E> collection) {
         return collection == null ? Collections.<E>emptyList() : collection;
     }
+    
+	public static String getInputFormNameByCollectionAndField(Collection collection, String field) throws DCInputsReaderException {
```

Comment:
I'm not sure we'll reuse this method elsewhere. Therefor I would move it to the DCInputsReader class and not make it `static`.

Reply:
Ok, I agree.